user/fonts-tex-gyre-bonum-math-otf/template.py

```diff
@@ -0,0 +1,14 @@
+pkgname = "fonts-tex-gyre-bonum-math-otf"
+pkgver = "1005"
+pkgrel = 0
+pkgdesc = "Math companion for TeX Gyre Bonum"
+license = "LPPL-1.3c"
```

Comment:
apparently the license is strictly speaking the GFL, which is "legally equivalent" to this one but with an extra non-legally binding clause

i don't know if it really matters that much, but maybe we specifically mention that one (ditto for the other templates too)?

```suggestion
license = "custom:GFL"
```